sexpress/src/main/scala/org/ensime/sexp/util/ThreadLocalSupport.scala

```diff
@@ -0,0 +1,7 @@
+package org.ensime.sexp.util
+
+trait ThreadLocalSupport {
```

Comment:
to be more idiomatically scala, it should be an `object`, not a `protected` mixin.